tools/storage.py

```diff
@@ -0,0 +1,173 @@
+import os
```

Comment:
I'd recommend moving this file under something like `rptest/tools` or something. Since it isn't used anywhere else.

Reply:
The reason i didn't is because it is more of a like a general utility then a specific thing just for testing. Most developers used it from vtools pointing it to a log directory

Reply:
Oh, I'm unaware of the backstory so definitely feel free to ignore the comment.
We could add a shebang `#!/usr/bin/env python3` for easier usage :thinking:

Reply:
sweet will do !